src/components/Navigation/TopBar/Help.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { FC, useState } from 'react';
+
+import { Button, Icon, Typography } from '@equinor/eds-core-react';
+import {
+  external_link,
+  file_description,
+  help_outline,
+} from '@equinor/eds-icons';
+import { tokens } from '@equinor/eds-tokens';
+
+import TopBarMenu from './TopBarMenu';
+
+import styled from 'styled-components';
+
+const { colors, spacings } = tokens;
+
+const MenuLinkItem = styled.a`
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+  padding: 16px 8px;
+  text-decoration: none;
+  gap: ${spacings.comfortable.medium};
+
+  &:hover {
+    background-color: #f7f7f7;
+  }
+`;
+
+const ContentInfo = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: ${spacings.comfortable.medium_small};
+  padding-right: 50px;
+`;
+
+export interface HelpProps {
+  applicationName: string;
+}
+
+export const Help: FC<HelpProps> = ({ applicationName }) => {
+  const [anchorEl, setAnchorEl] = useState<HTMLButtonElement | null>(null);
```

Comment:
Do we need to use anchorEl? What does it do? (I am actually just wondering cause I removed it on the other menus without a seeming effect :D )

Reply:
I think 🤔 we need it, what it does is that it says what element the menu should be placed around, so if you say `anchorEl = this button yellow button` and `placement = 'bottom-start'` on the `Menu` from EDS it should place the menu like this:
<img width="845" alt="image" src="https://github.com/equinor/amplify-components/assets/24507952/aeeb54e4-13ca-4d10-886c-0c2eec0f4cfe">